Makefile

```diff
@@ -33,14 +35,18 @@ binary: | $(TARGET) ## Build release binary with cargo
 $(TARGET):
 	MACOSX_DEPLOYMENT_TARGET="10.11" cargo build --release
 
+target/%/release/$(TARGET): phony_explicit
+	MACOSX_DEPLOYMENT_TARGET="10.11" cargo build --release --target=$*
```

Comment:
This will cause the build to fail unless the user building it has both targets installed, correct? I'm not sure that's good, I think users should probably be able to build a binary that works just on their system if they don't intend to ship it anywhere. Otherwise we'd have to document this in our installation instructions.

Reply:
That's correct, I'd be open to suggestions on how we can write this Makefile so it supports choosing whether you just want a native or a cross-platform build.

Reply:
I've updated the Makefile to add a condition that only builds for multiple architectures if you set the MULTI_ARCH option

Reply:
Maybe just a `make universal` instead of `make app`? Would probably be easier.

Reply:
Modified the makefile :+1: